integ-ecms/integ-ecms-social/src/main/java/org/exoplatform/wcm/ext/component/activity/FileUIActivityBuilder.java

```
@@ -43,7 +43,11 @@ protected void extendUIActivity(BaseUIActivity uiActivity, ExoSocialActivity act
 
     // set data into the UI component of activity
     if (activity.getTemplateParams() != null) {
-      fileActivity.setUIActivityData(activity.getTemplateParams());
+      try {
+        fileActivity.setUIActivityData(activity.getTemplateParams());
+      } catch (RepositoryException e) {
+        LOG.error("can not set activity data ", e);
```

Why adding this try / catch block, IMO, we shouldn't catch it here to ensure that higher layers of services / UI components are aware of this error. (Avoid swallowing exceptions)

---

Okay

---

done